Dockerfile

```diff
@@ -1,14 +1,16 @@
 FROM haskell:8
 
-RUN git clone https://github.com/facebookincubator/duckling.git
```

Comment:
This allows to build a standalone container, not requiring to be inside a checked out source directory. Also it fetches the latest source.

This might not be what you need (e.g. if you're looking to setup a safer prod environment without deploying latest library), but it's the preferred way here as it aims at providing a config to get started.